spec/words_counted/deprecated_spec.rb

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+require_relative "../spec_helper"
+
+module WordsCounted
+  warn "Methods being tested are deprecated"
+
+  describe Counter do
+    let(:counter) do
+      tokens = WordsCounted::Tokeniser.new("one three three three woot woot").tokenise
+      Counter.new(tokens)
+    end
+
+    describe "#word_density" do
+      it "returns words and their density in percent" do
+        expected = [
+          ['three', 50.0], ['woot', 33.33], ['one', 16.67]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.